app/src/main/res/values-sw600dp-port/dimens.xml

```diff
@@ -200,4 +200,20 @@
   <!-- Previous Responses Item: Question View -->
   <dimen name="previous_responses_item_question_view_margin_start">140dp</dimen>
   <dimen name="previous_responses_item_question_view_margin_end">140dp</dimen>
+
+  <!-- General Button Item: Exploration Split View -->
+  <dimen name="general_button_item_exploration_split_view_margin_start">48dp</dimen>
+  <dimen name="general_button_item_exploration_split_view_margin_end">48dp</dimen>
+
+  <!-- General Button: Exploration View -->
```

Comment:
https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/986ffd25-9e86-42a2-b870-aa6e754799e4/specs/

Reply:
Do we need these values?

As this is a port res dir, so if they will always use the split view, the split resources will be used. I think we can remove these items!! Let me know if I miss anything here.

Reply:
Actually Sliptview depends on two things: (a.) Physical size of more than 7" and (b.) DragAndDrop/ImageRegionSelection interaction

It is independent of tablet or mobile or its configuration